test/auditlog/test_auditlog_list.py

```diff
@@ -14,8 +14,8 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
 from csm.core.blogic import const
-from csm.cli.command_factory import CommandFactory
-from csm.cli.command import Output
+from cortx.utils.cli_framework.command_factory import CommandFactory
```

Comment:
We will not use Utils for CLI as of now. This change is not required. @udayan-y21 Please confirm